chapters/traversal.md

```diff
@@ -1,83 +1,85 @@
 # Understand Jina Recursive Document Representation
 
-In Jina, each Document is represented as a recursive representation (tree).
+## General overview
 
-A rooted recursive representation has a root node and every node has X children.
-In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+If you add a text document or an image to the Jina database, Jina indexes it as a new document. Internally, each Document is represented as a rooted recursive representation (tree).
+
+The rooted recursive representation consists of a *root node* and several *child nodes*. In Jina, the root node is the document itself, while the *left* and *right* children are referred to as *chunks* and *matches* respectively. The image below illustrates a basic document structure that consists of a document (root node) as well as two child nodes (chunks and matches).
```

Comment:
The root node is the original document itself.

Reply:
I added that to the explanation